lapis2/src/main/kotlin/org/genspectrum/lapis/model/ModelAggregated.kt

```diff
@@ -0,0 +1,39 @@
+package org.genspectrum.lapis.model
+
+import org.genspectrum.lapis.response.AggregatedResponse
+import org.genspectrum.lapis.silo.And
+import org.genspectrum.lapis.silo.PangoLineageEquals
+import org.genspectrum.lapis.silo.SiloAction
+import org.genspectrum.lapis.silo.SiloClient
+import org.genspectrum.lapis.silo.SiloFilterExpression
+import org.genspectrum.lapis.silo.SiloQuery
+import org.genspectrum.lapis.silo.StringEquals
+import org.genspectrum.lapis.silo.True
+import org.springframework.stereotype.Component
+
+@Component
+class ModelAggregated(private val siloClient: SiloClient) {
```

Comment:
I think I'd call this `AggregatedModel`. It's the usual pattern to call it `...Model`, `...Controller`, etc._